example/32_batched_gemm_scale_softmax_gemm/run_batched_gemm_scale_softmax_gemm_permute.inc

```diff
@@ -0,0 +1,259 @@
+int run(int argc, char* argv[])
```

Comment:
Thanks for refactoring. Does the .inc file need a license?

Reply:
Yes it should. Will add it

Reply:
Added